### ISSUE TITLE
Request to update Selenium.WebDriver to version 4.14.1

### ISSUE THREAD

**Issue by AngusBethke** [Dreamescaper/DotNetSeleniumExtras.Core#10]
Hi there, a CVE was recently raised with regards to older versions of the Selenium.WebDriver package. The recommendation is to push to a version greater than 4.14.0. As the latest version is now 4.14.1 it makes sense to upgrade to that.

Any chance you'll be able to do this for DotNetSeleniumExtras.PageObjects.Core and DotNetSeleniumExtras.WaitHelpers? As you seem to be the most recently active individual maintaining this suite of packages.

```
<testsuite failures="1" errors="0" time="0" id="4"
    package="DotNetSeleniumExtras.PageObjects.Core:4.3.0" skipped="0" tests="1"
    timestamp="2023-10-23T12:51:19.782">
    <testcase classname="CVE-2023-5590"
        name="pkg:nuget/DotNetSeleniumExtras.PageObjects.Core@4.3.0">
        <failure
            message="cvssV3: HIGH, score: 7.5 (CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H)" />
        <system-out>NULL Pointer Dereference in GitHub repository seleniumhq/selenium prior to
            4.14.0.</system-out>
    </testcase>
</testsuite>
<testsuite failures="1" errors="0" time="0" id="5"
    package="DotNetSeleniumExtras.WaitHelpers:3.11.0" skipped="0" tests="1"
    timestamp="2023-10-23T12:51:19.782">
    <testcase classname="CVE-2023-5590" name="pkg:nuget/DotNetSeleniumExtras.WaitHelpers@3.11.0">
        <failure
            message="cvssV3: HIGH, score: 7.5 (CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H)" />
        <system-out>NULL Pointer Dereference in GitHub repository seleniumhq/selenium prior to
            4.14.0.</system-out>
    </testcase>
</testsuite>
```

Thanks in Advance!

**Comment by Dreamescaper** [comment]
I can update `DotNetSeleniumExtras.PageObjects.Core`, but I have no relation to `DotNetSeleniumExtras.WaitHelpers` package, so I cannot update that one

**Comment by AngusBethke** [comment]
Thanks a stack :) 
I'll see if I can reach out to whoever manages the wait helpers.

**Comment by Dreamescaper** [comment]
I have published an updated version 4.14.1
https://www.nuget.org/packages/DotNetSeleniumExtras.PageObjects.Core/4.14.1
